src/blockdata/constants.rs

```diff
@@ -19,7 +19,8 @@
 //! single transaction
 //!
 
-use std::default::Default;
+use core::default::Default;
+use alloc::vec::Vec;
```

Comment:
I believe it's possible to avoid bumping MSRV if `use` like this is conditional. Assuming it's fine to have diferent MSRV for different features (I think it's reasonable).